components/FormCreate.js

```diff
@@ -232,7 +231,7 @@ export default function FormCreate() {
             onChange={handleKeyPress}
           />
         </WrapperSearchBar>
-        {typingInSearchbar ?  (
+        {typingInSearchbar ? (
```

Comment:
you can still use && instead of ? ... : here. 
```suggestion
        {typingInSearchbar && (
```